ginga/rv/plugins/Preferences.py

```diff
@@ -268,6 +268,40 @@
 images are being loaded into a channel frequently (e.g., a low frequency
 video feed), it may be undesirable to create thumbnails for all of them.
 
+**General Preferences**
+
+The "Num Images" setting specifies how many images can be retained in
+buffers in this channel before being ejected.  A value of zero (0) means
+unlimited--images will never be ejected.  If an image was loaded from
+some accessible storage and it is ejected, it will automatically be
+reloaded if the image is revisited by navigating the channel.  A setting
+of 1 to 4 is typical.
```

Comment:
Hmm. On the contrary, I recommend my users to set it to unlimited. This is because in their use case, they need to create mosaic or modify the buffer (e.g., background subtraction plugin). In those cases, they might lose the mosaic or modified images if it is limited. I recommend a deeper discussion about when to set buffer limit to what, if feasible.

Reply:
Sounds like we should devote a section in the manual to this. I agree that it makes sense if you are working with modified images in memory, but for the user who is mostly looking through data they are loading from disk there is no downside and a very big upside to having a limited size--because after only a few large images the machine will become slow and grind to a halt as the memory fills up.